rl_bakery/engine/timing_data.py

```diff
@@ -0,0 +1,36 @@
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+import logging
+
+logger = logging.getLogger(__name__)
+
+@dataclass
+class TimingData:
```

Comment:
I don't know if this class is needed. The configs are set in the RL App and the function to compute the current run_id is a utility.